configure.ac

```diff
@@ -395,6 +395,7 @@ AC_SUBST(WITH_OPENSSL_LIB)
 WITH_LIBGCRYPT_INCLUDE=
 WITH_LIBGCRYPT_LIB=
 if test "$with_crypto" = libgcrypt ; then
+  AC_DEFINE(WITH_LIBGCRYPT, 1, [Build with libgcrypt instead of nss3 support?])
```

Comment:
Why the `instead of nss3 support?`? This could just be `Build with libgcrypt as the crypto backend`.

Reply:
Thanks, I will change.